src/app/subnets/views/SubnetDetails/StaticDHCPLease/StaticDHCPTable/StaticDHCPTable.tsx

```diff
@@ -14,43 +20,63 @@ const headers = [
     sortKey: "interface",
   },
   { content: "Usage", className: "usage-col", sortKey: "usage" },
-  { content: "Owner", className: "oner-col", sortKey: "owner" },
   { content: "Comment", className: "comment-col", sortKey: "comment" },
   { content: "Actions", className: "actions-col" },
 ] as const;
 
-const generateRows = (dhcpLeases: StaticDHCPLease[]) =>
-  dhcpLeases.map((dhcpLease) => {
+const generateRows = (
+  reservedIps: ReservedIp[],
+  setSidePanelContent: SetSidePanelContent
+) =>
+  reservedIps.map((reservedIp) => {
     return (
-      <tr key={dhcpLease.mac_address}>
-        <td>{dhcpLease.ip_address}</td>
-        <td>{dhcpLease.mac_address}</td>
+      <tr key={reservedIp.id}>
+        <td>{reservedIp.ip}</td>
+        <td>{reservedIp.mac_address || "—"}</td>
```

Comment:
Let's add the title attribute so that it's possible to see the entire value on hover.